src/connectedk8s/azext_connectedk8s/tests/latest/test_connectedk8s_scenario.py

```diff
@@ -5,98 +5,521 @@
+            logger.warning("Failed to extract helm executable" + str(e))
+            return
+
+    return install_location
+
+
+def install_kubectl_client():
+    # Return kubectl client path set by user
+    try:
+
+        # Fetching the current directory where the cli installs the kubectl executable
+        home_dir = os.path.expanduser('~')
+        kubectl_filepath = os.path.join(home_dir, '.azure', 'kubectl-client')
+
+        try:
+            os.mkdir(kubectl_filepath)
+        except FileExistsError:
+            pass
+
+        operating_system = platform.system().lower()
+        # Setting path depending on the OS being used
+        if operating_system == 'windows':
+            kubectl_path = os.path.join(kubectl_filepath, 'kubectl.exe')
+        elif operating_system == 'linux' or operating_system == 'darwin':
+            kubectl_path = os.path.join(kubectl_filepath, 'kubectl')
+        else:
+            logger.warning(f'The {operating_system} platform is not currently supported for installing kubectl client.')
+            return
+
+        if os.path.isfile(kubectl_path):
+            return kubectl_path
+
+        # Downloading kubectl executable if its not present in the machine
+        get_default_cli().invoke(['aks', 'install-cli', '--install-location', kubectl_path])
+        # Return the path of the kubectl executable
+        return kubectl_path
+
+    except Exception as e:
+        logger.warning("Unable to install kubectl. Error: " + str(e))
+        return
+
+
 class Connectedk8sScenarioTest(LiveScenarioTest):
 
-    def test_connectedk8s(self):
+    @live_only()
+    @ResourceGroupPreparer(name_prefix='conk8stest', location='eastus2euap', random_name_length=16)
+    def test_connect(self,resource_group):
+        
+        managed_cluster_name = self.create_random_name(prefix='test-connect', length=24)
+        kubeconfig="%s" % (_get_test_data_file(managed_cluster_name + '-config.yaml')) 
+        self.kwargs.update({
+            'rg': resource_group,
+            'name': self.create_random_name(prefix='cc-', length=12),
+            'kubeconfig': kubeconfig,
+            'managed_cluster_name': managed_cluster_name
+        })
+
+        self.cmd('aks create -g {rg} -n {managed_cluster_name} --generate-ssh-keys')
+        self.cmd('aks get-credentials -g {rg} -n {managed_cluster_name} -f {kubeconfig}')
+        self.cmd('connectedk8s connect -g {rg} -n {name} -l eastus --tags foo=doo --kube-config {kubeconfig} --kube-context {managed_cluster_name}', checks=[
+            self.check('tags.foo', 'doo'),
+            self.check('resourceGroup', '{rg}'),
+            self.check('name', '{name}')
+        ])
 
-        managed_cluster_name = self.create_random_name(prefix='cli-test-aks-', length=24)
+        self.cmd('connectedk8s delete -g {rg} -n {name} --kube-config {kubeconfig} --kube-context {managed_cluster_name} -y')
+        self.cmd('aks delete -g {rg} -n {managed_cluster_name} -y')
```

Comment:
there's no self.check here , will it still show up in test failures if this command fails here?